### PR TITLE
new release v0.12.16

### PR DESCRIPTION
**Pull request by arthurzam**

### CODE REVIEW

File: NEWS.rst [hunk 1]
```diff
@@ -2,6 +2,16 @@
...
 
+----------------------------
+pkgcore 0.12.16 (2022-10-04)
+----------------------------
+
+- portage_conf: fix timeout settings for rsync (#368, Arthur Zamarin)
+
+- fetch.custom: fix commands with `\`` variable escaping (Arthur Zamarin)
+
+- repository: check issues with distdir only if fetch failed (Arthur Zamarin)
```
Comment (thesamesam) [inline NEWS.rst:13]: I'd reference the pkgdev bug, even if it's only somewhat related.

File: NEWS.rst [hunk 2]
```diff
@@ -2,6 +2,16 @@
 Release Notes
 =============
 
+----------------------------
+pkgcore 0.12.16 (2022-10-04)
+----------------------------
+
+- portage_conf: fix timeout settings for rsync (#368, Arthur Zamarin)
+
+- fetch.custom: fix commands with `\`` variable escaping (Arthur Zamarin)
```
Comment (thesamesam) [inline NEWS.rst:11]: Reference the pkgdev bug?

**Review by thesamesam — APPROVED** [review]

File: NEWS.rst [hunk 3]
```diff
@@ -2,6 +2,18 @@
...
```
Comment (thesamesam) [inline NEWS.rst:11]: wait, sorry, is double ` like this intended?

Comment (arthurzam) [reply NEWS.rst:11]: No, thanks for catching it

**Pull request merged**